.travis.yml

```diff
@@ -34,38 +34,23 @@ matrix:
       <<: *linux-setup-steps
       script: pipenv run -v test -f make,ninja
 
-    - name: "Linux: test with make and ninja on Python 2.7"
-      python: '2.7'
-      <<: *linux-setup-steps
-      script: pipenv run -v test -f make,ninja
-
-    - name: "Windows: test with mocks on Python 3"
-      <<: *windows-setup-steps
-      env:
-        - PATH=/c/Python37:$PATH
-      before_install: choco install python3
-      script: python -m pipenv run -v test -f make-mock,msvs-mock
-
-    - name: "Windows: test with mocks on Python 2"
-      <<: *windows-setup-steps
-      env:
-        - PATH=/c/Python27:$PATH
-      before_install: choco install python2
-      script: python -m pipenv run -v test -f make-mock,msvs-mock
-
     - name: "macOS: test with make and ninja on Python 3.7"
       <<: *osx-setup-steps
       script: pipenv run -v test -f make,ninja
 
-    - name: "lint with Python 2.7"
-      python: '2.7'
-      <<: *linux-setup-steps
-      script: pipenv run -v lint
-
-    - name: "lint with Python 3.7"
-      python: '3.7'
-      <<: *linux-setup-steps
-      script: pipenv run -v lint
+    - name: "macOS: XCode10.2 test with make and ninja on Python 3.7"
+      os: osx
+      osx_image: xcode10.2  # provides Python 3.7.2
+      language: cpp  # 'language: python' is not yet supported on macOS
+      # before_install: brew upgrade python  # takes too long
```

Comment:
Lose the comment.  We don't need to upgrade Python because we get similar results by upgrading xcode instead.

Reply:
Done